sdk/storage/mgmt/src/main/java/com/azure/management/storage/implementation/BlobContainersImpl.java

```diff
@@ -54,7 +54,7 @@ private ImmutabilityPolicyImpl wrapImmutabilityPolicyModel(ImmutabilityPolicyInn
         return new ImmutabilityPolicyImpl(inner, manager());
     }
 
-    private Mono<ImmutabilityPolicyInner> getImmutabilityPolicyInnerUsingBlobContainersInnerAsync(String id) {
+    public Mono<ImmutabilityPolicyInner> getImmutabilityPolicyInnerUsingBlobContainersInnerAsync(String id) {
```

Comment:
why public?

Reply:
It isn't used as a private function. If we would not make it public / protected, we need to delete it. Since the impl is not public, I think it ok to public the function in order for future usage.